src/bidders/a9/index.js

```diff
@@ -64,6 +64,12 @@ export class A9 extends BaseBidder {
 			a9Slots = a9Slots.concat(this.slotsVideo.map(this.createVideoSlotDefinition));
 		}
 
+		const disabledSlots = context.get('bidders.disabledSlots');
+
+		if (disabledSlots) {
```

Comment:
`slot-service` already has kind of slot "states". It isn't exposed. But maybe we should is it here? Let's don't block this pull request... but we can add some todo note for later :(

Reply:
This PR is already quite big. I think it's better to do this in different commit, separate task (also this change can be quite majorish).